ewfs/scenario.py

```diff
@@ -119,9 +119,10 @@ def circuit(self) -> QuantumCircuit:
         )
 
         # Adding classical flag registers and measurements of the flag qubits here if GHZ setting is used.
+
         if self.friend_state == GHZ:
-            creg_debbie_flag_qubits = ClassicalRegister(len(self.debbie_flag_qubits), "debbie_flag_qubits")
-            creg_charlie_flag_qubits = ClassicalRegister(len(self.charlie_flag_qubits), "charlie_flag_qubits")
+            creg_debbie_flag_qubits = ClassicalRegister(len(self.debbie_flag_qubits), 'debbie_flag_qubits')
+            creg_charlie_flag_qubits = ClassicalRegister(len(self.charlie_flag_qubits), 'charlie_flag_qubits')
```

Comment:
[nit]: If you want to maintain the "charlie->debbie" order for consistency as you do below, we could do 

(also changing single to double quotes for consistency).

```suggestion            
            creg_charlie_flag_qubits = ClassicalRegister(len(self.charlie_flag_qubits), "charlie_flag_qubits")
            creg_debbie_flag_qubits = ClassicalRegister(len(self.debbie_flag_qubits), "debbie_flag_qubits")
```